### PR TITLE
Updating ICESat-2 Resources page

### PR DESCRIPTION
**Pull request by zachghiaccio**
Updating ICESat-2 resources to add other githubs that process ICESat-2 data.

### CODE REVIEW

**Comment by JessicaS11** [comment]
@zachghiaccio can you remind me what this PR still needs so we can convert it from draft and review?

**Comment by zachghiaccio** [comment]
I just looked through, and I think it should be ready for review.

File: doc/source/community/resources/2022_IS2_HW.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.. _resource_IS2HW_2022:
+
+Third ICESat-2 Hackweek Facilitated by the University of Washington eScience Institute
+--------------------------------------------------------------------------------------
+This event is currently being planned, and will take place 21-25 March 2022.
+Please see the `event website <https://icesat-2hackweek.github.io/>`_ for more information.
```
Comment (JessicaS11) [inline doc/source/community/resources/2022_IS2_HW.rst:6]:
```suggestion
Please see the `event website <https://icesat-2.hackweek.io/>`_ for more information.
```

Comment (zachghiaccio) [reply doc/source/community/resources/2022_IS2_HW.rst:6]: Are there any plans for the former hackweek url? I've been going to that one by mistake over the past week.

**Review by kelseybisson — APPROVED** [review]
Looks good to me!

**Pull request merged**